### ISSUE TITLE
Remove "github/workspace" from filename in csv?

### ISSUE THREAD

**Issue by maelle** [urlstechie/urlchecker-action#56]
Not sure if this belongs to the library or action. Would it be possible (and make sense) to have the filename relative to the folder in which the URL checks are run?

**Comment by vsoch** [comment]
That is derived programatically with `os.path.abspath()` - so if you give a relative path and the github workspace is the cwd, this is what you'd expect to see. So, it should be relative. The issue I think you are seeing is that the action is run inside a container, where /github/workspace != the same bound path on the host. So the container sees /github/workspace/results.csv, but your local host (running the action) if the next step isn't in another container would see something in /home. Can you give a specific example of where you ran into an issue so I can take a look?

**Comment by maelle** [comment]
In https://github.com/r-hub/docs/actions/runs/77341786 and workflow is at https://github.com/r-hub/docs/actions/runs/77341786/workflow :-) Thank you!

**Comment by vsoch** [comment]
hey @maelle ! So you can see that the save path is passed without any subfolder /absolute path:
![image](https://user-images.githubusercontent.com/814322/79200272-6a2bc600-7df3-11ea-8fe2-74d434b8454c.png)

The /github/workspace as I mentioned previously is derived when we do os.path.abspath() of that. In the container that is bound to the host at /home/runner/work/docs/docs to /github/workspace, take a look at the end of the line here:

![image](https://user-images.githubusercontent.com/814322/79200365-8fb8cf80-7df3-11ea-9c34-444dc2841056.png)

So I'm not sure what action you are looking to take? This bind of /github/workspace to that folder on the host is an actions convention - that is how the container accesses files.

**Comment by vsoch** [comment]
Are you wanting to just remove it for aesthetic reasons? We could have the checker derive a relative path for printing and saving. @SuperKogito do you want to weigh in?

**Comment by vsoch** [comment]
Just to clarify instead of:

![image](https://user-images.githubusercontent.com/814322/79200667-0655cd00-7df4-11ea-8043-1be0b7270e02.png)

You want to see:

![image](https://user-images.githubusercontent.com/814322/79200737-238a9b80-7df4-11ea-9849-d6ed3cd6197f.png)

**Comment by maelle** [comment]
It is fine if it stays, I just thought it was not informative in the csv file (I was expecting the filename column to contain file paths relative to the folder root)

**Comment by maelle** [comment]
I think there was a misunderstanding, the log is fine, I was asking about the content of the csv file 🙂

**Comment by maelle** [comment]
(That said even in the log it'd be nicer with a shorter path as in your screenshot)

**Comment by vsoch** [comment]
Yes I understand now! I'm actually in agreement - I think the contents of the csv would do well to have relative paths, but I think it's most useful to leave the full paths for the print to the screen. This is an issue for urlchecker-python so I'm going to transfer there.
